meta-resin-common/conf/distro/include/resin-os.inc

```diff
@@ -74,3 +74,4 @@ RESIN_FLASHER_FLAG_FILE = "resin-image-flasher"
 INITRAMFS_IMAGE = "resin-image-initramfs"
 INITRAMFS_IMAGE_BUNDLE = "1"
 KERNEL_INITRAMFS = "-initramfs"
+INITRAMFS_TASK = ""
```

Comment:
@telphan you don't need to paste code in the commit log but it would be nice to have an explanation which can say something similar to: this used to be the way that yocto was doing that until version whatever where they changed it to that. Some BSP might still use variable X as in the case of Y - link.